pkg/apis/pipeline/v1/pipeline_types_test.go

```diff
@@ -107,7 +107,123 @@ func TestPipelineTask_ValidateRefOrSpec(t *testing.T) {
 		},
 		expectedError: &apis.FieldError{
 			Message: `expected exactly one, got both`,
```

Comment:
please fix this error in the validation routine:

`expected exactly one, got both`

Its now received two out of four available options.